src/js/index.js

```diff
@@ -1,2 +1,49 @@
 import 'bootstrap/dist/css/bootstrap';
-import $ from 'jquery';
+
+class BitconFetcher {
+  constructor(location) {
+    this.target = location;
```

Comment:
`tagretClass` is make more sense

Reply:
It's a Id string, not a class name. So the parameter should be better to use `targetContainerId` or something similar.